modules/cloud-init/templates/proxy.sh.tmpl

```diff
@@ -0,0 +1,7 @@
+export http_proxy="${proxy_url}"
+export https_proxy="${proxy_url}"
+%{ if repl_cidr == "" ~}
+export no_proxy=10.0.0.0/8,127.0.0.1,169.254.169.254
+% { else ~}
+export no_proxy=10.0.0.0/8,127.0.0.1,169.254.169.254,${repl_cidr}
+%{ endif ~}
```

Comment:
For maintainability, so we don't have to change two high-density variables could we change this to basically leverage this setup?

```suggestion
export no_proxy=10.0.0.0/8,127.0.0.1,169.254.169.254
%{ if repl_cidr != "" ~}
export no_proxy=${no_proxy},${repl_cidr}
%{ endif ~}
```